nautobot/ui/src/components/Retrieve/RenderHeader.js

```diff
@@ -21,47 +22,63 @@ import { humanFriendlyDate } from "@utils/date";
 
 export default function RenderHeader({ data }) {
     return (
-        <Box display="flex" justifyContent="space-between" padding="md">
-            <Heading display="flex" alignItems="center" gap="5px">
+        <Flex justify="space-between">
```

Comment:
Should we be using `Flex` instead of `Box` from now on? What are the differences?

Reply:
The difference between `Flex` and `Box` is esentially that `Box` has, by default, `display: block;`, whereas `Flex` has `display: flex;`. Outside (i.e. for their immediate parents) `block` and `flex` are the same, but inside, their children flow is fundamentally different. MDN can offer more detailed explanation: https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox